src/main/fc/fc_tasks.c

```diff
@@ -57,36 +57,43 @@ cfTask_t cfTasks[] = {
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_ATTITUDE] = {
+        .taskName = "ATTITUDE",
+        .taskFunc = taskUpdateAttitude,
+        .desiredPeriod = 1000000 / 100,
```

Comment:
yeah, there's a few other places which would benefit from using similar macros.  and i agree the comments are bad.